pkg/spdk/engine.go

```diff
@@ -648,7 +648,12 @@ func (e *Engine) checkAndUpdateInfoFromReplicaNoLock() {
 		}
 		replica, err := replicaServiceCli.ReplicaGet(replicaName)
 		if err != nil {
-			e.log.WithError(err).Warnf("failed to get replica %s with address %s, will skip this replica and continue info update from replica", replicaName, address)
+			if strings.Contains(err.Error(), "cannot find replica") {
+				e.log.WithError(err).Warnf("Failed to get replica %s with address %s, make it as ERR", replicaName, address)
+				e.ReplicaModeMap[replicaName] = types.ModeERR
+			} else {
+				e.log.WithError(err).Warnf("failed to get replica %s with address %s, will skip this replica and continue info update from replica", replicaName, address)
```

Comment:
NIT: Correct the printed error log format then it's good to go

Reply:
Fixed. Thank you.